svix-cli/src/config.rs

```diff
@@ -0,0 +1,86 @@
+use anyhow::Result;
+use figment::{
+    providers::{Env, Format, Toml},
+    Figment,
+};
+use serde::{Deserialize, Serialize};
+use std::io::Write;
+use std::os::unix::fs::OpenOptionsExt;
+use std::path::{Path, PathBuf};
+
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct Config {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub auth_token: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    server_url: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    debug_url: Option<String>,
+
+    // Relay stuff relates to the `listen` command.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub relay_token: Option<String>,
+    // FIXME: "url" isn't right. We expect a hostname, default is: `api.play.svix.com`
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub relay_debug_url: Option<String>,
```

Comment:
Rename the field and use `#[serde(alias)]` or `#[serde(rename)]` to keep backwards compat?

Reply:
`relay_hostname`? Sure. Why not fix it now, eh?